.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This file should not be comitted in

Reply:
Done.  I've also added .vscode to the gitignore